packages/components/src/InputEmail/InputEmail.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { FormField, FormFieldProps } from "../FormField";
+
+type InputEmailProps = Pick<
+  FormFieldProps,
+  Exclude<
+    keyof FormFieldProps,
+    | "children"
+    | "autocomplete"
+    | "max"
+    | "min"
+    | "rows"
+    | "type"
+    | "keyboard"
+    | "onEnter"
+    | "onBlur"
+    | "inputRef"
+    | "actionsRef"
+    | "onFocus"
+  >
```

Comment:
What's the reason we want to exclude `onEnter `, `onFocus`, `onBlur`, `autocomplete`, `inputRef`?

Reply:
We don't require it in this component.  We want to expose as little as possible.